css/clean-blog.css

```diff
@@ -409,190 +409,3 @@ footer .copyright {
   font-size: 16px;
   padding: 25px 35px;
 }
-
```

Comment:
Please check if this code is necessary. I removed it by accident when updating library versions